cpp/oneapi/dal/policy.hpp

```diff
@@ -39,25 +40,47 @@ enum class cpu_extension : uint64_t {
     avx512 = 1U << 5
 };
 
-class ONEAPI_DAL_EXPORT default_execution_context : public base {
+class ONEAPI_DAL_EXPORT default_policy : public base {
```

Comment:
I think that for DPC++ world `default_policy` is a policy with a queue created from `default_selector`. Suggest to make its name like in data management, `host_seq_policy`, or `host_par_policy`

Reply:
We still need to discuss `seq` and `par` policies, because they slightly contradicts with the approach with two threading layers `daal_thread.so`/`daal_sequential.so`. User selects threading layer at link time (when they link their application), initial DAAL design assumes user don't need to change their code to use sequential or parallel version.

Reply:
You are right, maybe `host_policy` is more suitable name.